compiler-rt/lib/radsan/radsan_context.cpp

```diff
@@ -25,10 +26,21 @@ static pthread_key_t key;
 static pthread_once_t key_once = PTHREAD_ONCE_INIT;
 void internalFree(void *ptr) { InternalFree(ptr); }
 
+static __sanitizer::atomic_uint64_t radsan_report_count{};
+
+void IncrementReportCount();
 } // namespace detail
 
 namespace radsan {
 
+__sanitizer::u64 GetReportCount() { 
+  return __sanitizer::atomic_load(&detail::radsan_report_count, memory_order_acquire); 
+}
+
+void IncrementReportCount() { 
```

Comment:
Moved these report methods into this file, as it's where they are used. It may make sense to make a `radsan_report.cpp/.h`, especially once we get an exit report if we do that with the `continue` mode.